### PR TITLE
Add support for Debian 10

### PR DESCRIPTION
**Pull request by thechristschn**
There ist currently no package for a puppet5 puppetserver on Debian 10, so I added travis tests only for puppet6.

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Thank you for the contribution!

File: .travis.yml [hunk 1]
```diff
@@ -84,6 +84,12 @@ matrix:
     env: BEAKER_set="debian-9" BEAKER_PUPPET_COLLECTION=puppet6
     bundler_args:
     script: bundle exec rake beaker
+  - rvm: 2.5.3
```
Comment (ghoneycutt) [inline .travis.yml]: please add puppet 5

File: templates/sshd.debian10.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+session    optional     pam_keyinit.so force revoke
+@include common-session
+session    optional     pam_motd.so  motd=/run/motd.dynamic
+session    optional     pam_motd.so noupdate
+session    optional     pam_mail.so standard noenv # [1]
+session    required     pam_limits.so
+session    required     pam_env.so # [1]
+session    required     pam_env.so user_readenv=1 envfile=/etc/default/locale
+session [success=ok ignore=ignore module_unknown=ignore default=bad]        pam_selinux.so open
+
```
Comment (ghoneycutt) [inline templates/sshd.debian10.erb]: please remove trailing blank line

**Comment by ghoneycutt** [comment]
Are these configs from a standard install without any other configuration added?

File: .travis.yml (hunk 1 above)
Comment (thechristschn) [reply .travis.yml]: With which rvm version? There is no puppetserver5 package for debian 10 available.

File: templates/sshd.debian10.erb (hunk 2 above)
Comment (thechristschn) [reply templates/sshd.debian10.erb]: Fixed.

**Comment by thechristschn** [comment]
> Are these configs from a standard install without any other configuration added?

This is the default config from a new standard installation. The only change I made was to remove `@include common-password` from the end of `templates/sshd.debian10.erb` . But this line was already removed for debian9 from the standard installation, so I guess there is a reason. But I can readd it if you want.

File: .travis.yml (hunk 1 above)
Comment (ghoneycutt) [reply .travis.yml]: It doesn't use the puppetserver, it uses the agent package.

**Comment by ghoneycutt** [comment]
Please add that line back so we can keep it as default as possible. Thank you!

**Comment by thechristschn** [comment]
> Please add that line back so we can keep it as default as possible. Thank you!

Done.

**Comment by ghoneycutt** [comment]
Vagrant is failing

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

apt-get install -y linux-headers-`uname -r` dkms

Stdout from the command:

Reading package lists...
Building dependency tree...
Reading state information...


Stderr from the command:

E: Unable to locate package linux-headers-4.19.0-5-amd64
E: Couldn't find any package by glob 'linux-headers-4.19.0-5-amd64'
E: Couldn't find any package by regex 'linux-headers-4.19.0-5-amd64'
```

**Comment by ghoneycutt** [comment]
Vagrant is working. Great work!

**Pull request merged**